waypoints_y: [-1262.96, -1272.61, -1278.33, -1285.09, -1288.61, -1299.17, -1311.12, -1327.05, -1345.09, -1356.09, -1363.63, -1370.9, -1371.44, -1371.07, -1372.53, -1374.05, -1376.23, -1378.06, -1379.97, -1382.12, -1383.97, -1388.34, -1393.88, -1399.63, -1412.05, -1424.12, -1434.49, -1446.33, -1459.58, -1470.62, -1481.12, -1490.57, -1498.72, -1507.44, -1516.08, -1523.4, -1530.73, -1538.34, -1548.99, -1556.77, -1564.74, -1573.17, -1581.09, -1589.95, -1599.36, -1605.44, -1611.57, -1620.11, -1628.46, -1635.03, -1639.04, -1639.33, -1639.08, -1638.66, -1638.3, -1638.19, -1638.09, -1639.95, -1643.46, -1648.96, -1656.56, -1666.91, -1676.82, -1686.02, -1697.92, -1711.86, -1725.4, -1738.81, -1749.14, -1754.81, -1756.1, -1756.86, -1757.62]
waypoints_x: [366.564, 370.708, 374.901, 384.274, 392.485, 402.745, 411.952, 412.849, 408.382, 403.916, 397.799, 388.778, 372.05, 356.011, 339.981, 325.038, 307.911, 292.954, 277.642, 259.424, 243.886, 231.982, 219.474, 211.949, 206.894, 202.161, 197.934, 192.06, 184.482, 178.532, 172.65, 165.619, 158.374, 151.608, 143.968, 137.409, 129.561, 123.908, 118.711, 116.908, 117.77, 120.973, 125.175, 130.359, 135.993, 139.789, 143.397, 148.338, 153.433, 160.231, 168.851, 179.712, 191.31, 201.466, 212.807, 225.333, 238.088, 249.471, 260.619, 270.402, 279.459, 290.845, 301.249, 310.681, 323.437, 339.302, 352.334, 365.858, 376.378, 387.029, 400.093, 417.044, 429.51]